--TEST--
Bug #36268 (Object destructors called even after fatal errors)
--FILE--
<?crx
class Foo {
    function __destruct() {
        echo "Ha!\n";
    }
}
$x = new Foo();
bar();
?>
--EXPECTF--
Fatal error: Uncaught Error: Call to undefined function bar() in %sbug36268.crx:8
Stack trace:
#0 {main}
  thrown in %sbug36268.crx on line 8
Ha!
